.env.sample

```diff
@@ -2,4 +2,14 @@ DATABASE_USERNAME=root
 DATABASE_PASSWORD=
 DATABASE_HOST=127.0.0.1
 DATABASE_NAME=stocks_trading_app
-DATABASE_NAME_TEST=stocks_trading_app_test
\ No newline at end of file
+DATABASE_NAME_TEST=stocks_trading_app_test
+
+MAILER_EMAIL=
+MAILER_PASSWORD=
+
+DATABASE_ADAPTER=
```

Comment:
useless since adapter is already hardcoded sa database.yml.. Pwede mong idelete to